openslides_backend/action/actions/motion/create_forwarded.py

```diff
@@ -65,18 +66,35 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 ]
                 self.execute_other_action(UserUpdate, user_update_payload)
         else:
+            username = committee.get("name", "Committee User")
             committee_user_create_payload = {
-                "username": committee.get("name", "Committee User"),
+                "username": username,
```

Comment:
You can just remove this line and the whole while-loop below, because without a username given, `user.create` will automatically generate one by appending a number.